app/graph/types/project_media_type.rb

```diff
@@ -152,6 +152,14 @@
     }
   end
 
+  field :translations_count do
+    type types.Int
+
+    resolve ->(project_media, _args, _ctx) {
+      project_media.get_annotations('translation').count
+    }
+  end
```

Comment:
Could it be a generic GraphQL field that receives the annotation type as a parameter?